curriculum/challenges/english/20-upcoming-python/learn-string-manipulation-by-building-a-cipher/6555e43e783ed31a0532b1b2.md

```diff
@@ -0,0 +1,64 @@
+---
+id: 6555e43e783ed31a0532b1b2
+title: Step 89
+challengeType: 20
+dashedName: step-89
+---
+
+# --description--
+
+The newline character `\n` is a special sequence used to represent a new line. You can use it as a normal character inside your strings.
```

Comment:
```suggestion
The newline character `\n` is a special sequence used to represent a new line. You can write a backslash `\` followed by an `n` as a normal sequence of characters in a string and it will be replaced by a new line in the output when the program runs.
```

Reply:
I would suggest specifying that this sequence of characters will be replaced by a new line in the output and that the sequence \n will not be displayed in the final string.